packages/core-api/lib/versions/2/handlers/transactions.js

```diff
@@ -40,17 +40,12 @@ exports.store = {
    * @return {Hapi.Response}
    */
   async handler (request, h) {
-    const blockchain = container.resolvePlugin('blockchain')
-    if (!transactionPool || !blockchain) {
+    if (!transactionPool) {
```

Comment:
Actually this whole block can be removed, the API will only be mounted after the pool and blockchain are ready.